src/proof/proof_manager.cpp

```diff
@@ -1063,5 +1060,15 @@ void ProofManager::printGlobalLetMap(std::set<Node>& atoms,
 void ProofManager::ensureLiteral(Node node) {
   d_cnfProof->ensureLiteral(node);
 }
-
+void ProofManager::printTrustedTerm(Node term,
+                                    std::ostream& os,
+                                    ProofLetMap& globalLetMap)
+{
+  if (ProofManager::currentPM()->getTheoryProofEngine()->printsAsBool(term))
```

Comment:
Suggestion: Instead of calling `ProofManager::currentPM()` multiple times, maybe just call it in the beginning of the function and store it in a `ProofManager*`? Similar to what we usually do with our `NodeManager`.